src/org/opensolaris/opengrok/configuration/Configuration.java

```diff
@@ -682,8 +739,18 @@ public String getBugPage() {
         return bugPage;
     }
 
-    public void setBugPattern(String bugPattern) {
-        this.bugPattern = bugPattern;
+    /**
+     * Set the bug pattern to a new value
+     *
+     * @param bugPattern the new pattern
+     * @throws PatternSyntaxException when the pattern is not a valid regexp or
+     * does not contain at least one capture group
+     */
+    public void setBugPattern(String bugPattern) throws PatternSyntaxException {
+        if (!bugPattern.matches(".*\\([^\\)]*\\).*")) {
```

Comment:
should this be more strict ? i.e. require at least one character inside the parens ?